src/npcmove.cpp

```diff
@@ -3711,14 +3723,18 @@ void npc::use_painkiller()
 static float rate_food( const item &it, int want_nutr, int want_quench )
 {
     const auto &food = it.get_comestible();
+
+    // Don't eat it if it's not food.
```

Comment:
Commenting code that is self-explanatory doesn't really help, some of your comments explain simple one-line checks